### PR TITLE
Server Changes

### PR DESCRIPTION
**Pull request by thenav56**
TODO:
- [x] Ary: Clone Ary to other projects the-deep/deeper#898
- [x] Entry comments: Make assignee in entry comments multiselect the-deep/deeper#894
- [x] Organigram Widget: Show parents of selected nodes in new columns the-deep/deeper#897
  - NOTE: Will require manual migration for old organigram data. cc: @AdityaKhatri

### CODE REVIEW

File: apps/lead/views.py [hunk 1]
```diff
@@ -651,57 +761,6 @@ def _get_clone_ready(obj, lead):
             _get_clone_ready(emm_trigger, lead) for emm_trigger in emm_triggers
         ])
 
+        if return_existing_lead:
+            return lead, True
         return lead
```
Comment (bewakes) [inline apps/lead/views.py]: There are two different return types. Might cause issues at the calling end.

**Review by bewakes — COMMENTED** [review]
All is good, except for one place where a function has two different return types(one tuple and the other object).

File: apps/lead/views.py (hunk 1 above)
Comment (thenav56) [reply apps/lead/views.py]: @bewakes Thanks for the review. Could you check out this commit https://github.com/the-deep/server/pull/476/commits/7d0aca86e85076f59ae6bfdcc31e9f12ae72e9dd, if it resolves?

**Review by bewakes — APPROVED** [review]

File: apps/lead/views.py (hunk 1 above)
Comment (bewakes) [reply apps/lead/views.py]: Perfect!

**Pull request merged**